src/file/FileAppendTransaction.js

```diff
@@ -445,6 +457,90 @@ export default class FileAppendTransaction extends Transaction {
         return "fileAppend";
     }
 
+    /**
+     * Build all the transactions when transacgtions are not signed
+     * @override
+     * @internal
+     */
+    _buildIncompletedTransactions() {
```

Comment:
nit: *Incomplete

Reply:
Hm, I changed it but now that I think about it shouldn't it be `Incompleted`? :D